C22_ROW_TYPE[] row